### ISSUE TITLE
`libtcmalloc_minimal.4.dylib` not found on macOS

### ISSUE THREAD

**Issue by matthesjh** [SWI-Prolog/swipl-devel#570]
When starting `swipl` version 8.1.27 on macOS (10.15.4 build 19E266) the following error occurs.

```
dyld: Library not loaded: /opt/local/lib/libtcmalloc_minimal.4.dylib
  Referenced from: /Applications/SWI-Prolog.app/Contents/MacOS/swipl
  Reason: image not found
```

This problem is not present in earlier versions.

**Comment by pmoura** [comment]
If you have MacPorts installed:

```bash
$ sudo port install gperftools
```

**Comment by matthesjh** [comment]
I used the application bundle from the download page. So everything should work without installing any additional tools or libraries, just like the older versions, because there is no notice to install some additional tools or libraries.

**Comment by JanWielemaker** [comment]
Yes.  Something went wrong creating the bundle.   Will be fixed, but might take a couple of days.  In the meanwhile @pmoura's fix should work and swipl-win (the real app) probably also works.

**Comment by matthesjh** [comment]
I'll close this, because it was fixed with the release of version 8.1.28.
